src/app/src/components/QuizNavbar.js

```diff
@@ -4,23 +4,93 @@ import styled from 'styled-components';
 import { Box, Flex } from 'rebass';
 import { Heading, Button, buttonWidth } from './custom-styled-components';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { themeGet } from 'styled-system';
 
 import { hideQuiz } from '../actions';
 import { QuizResult } from '../util/QuizResult';
 
 const StyledQuizNavbar = styled(Flex)`
-    display: flex;
-    align-items: center;
     justify-content: space-between;
+    position: sticky;
+    top: 0;
+    z-index: 1;
     width: 100%;
-    height: 50px;
-    background-color: #296882;
-    padding: 0 10px 0 30px;
+    align-items: center;
+    padding: 0 ${themeGet('space.comfortable')};
+    height: ${themeGet('navHeight')};
+    border-bottom: 1px solid rgba(256, 256, 256, 0.2);
+    background: linear-gradient(
+        -90deg,
+        ${themeGet('colors.teals.1')} 0%,
+        ${themeGet('colors.teals.2')} 100%
+    );
 `;
 
 const Result = styled(Box)`
     flex: none;
-    border-bottom: ${props => (props.current ? '1px solid green' : null)};
+    padding: 0 1rem;
+    paint-order: stroke;
+    stroke-width: ${props => props.theme.sw};
+    stroke: ${props => props.theme.s};
+    color: ${props => props.theme.c};
+    transition: stroke-width 0, stroke 0.35s ease-out, color 0.35s ease-out;
+
+    .fa-fish {
+        stroke-width: ${props => props.theme.sw};
+        stroke: ${props => props.theme.s};
+        color: ${props => props.theme.c};
+        transition: stroke-width 0, stroke 0.35s ease-out, color 0.35s ease-out;
+    }
+`;
+
+const theme = {
+    sw: null,
+    s: null,
+    c: '#052231',
+};
+
+const currentTheme = {
+    sw: '50px',
+    s: '#CEE007',
+    c: '#052231',
+};
+
+const correctTheme = {
+    sw: '50px',
+    s: 'rgba(49, 86, 105, 0.8)',
+    c: 'rgba(191, 204, 94, 0.4)',
+};
+
+const incorrectTheme = {
+    sw: '50px',
+    s: 'rgba(49, 86, 105, 0.8)',
+    c: 'rgba(255, 255, 255, 0.38)',
+};
+
+const CorrectIcon = styled(Box)`
+    position: relative;
+
+    .icon {
+        color: ${themeGet('colors.greens.2')};
+        stroke-width: 0;
+        position: absolute;
+        top: -0.3rem;
+        left: 57%;
+        transform: translateX(-30%);
+    }
+`;
+
+const IncorrectIcon = styled(Box)`
```

Comment:
Consider refactoring shared css from `CorrectIcon` and `IncorrectIcon` in an `Icon` base styled component